webapp/src/DOMJudgeBundle/Service/RejudgingService.php

```diff
@@ -138,6 +138,18 @@ public function finishRejudging(Rejudging $rejudging, string $action, callable $
 
         $this->dj->auditlog('rejudging', $rejudgingId, $action . 'ing rejudge', '(start)');
 
+        // Add missing state events for this contest. We do this here and disable doing it in the
+        // loop when calling EventLogService::log, because then we would do the same check a lot of
+        // times, which is really inefficient and slow. Note that it might be the case that a state
+        // change event will happen exactly during applying a rejudging *and* that no client is
+        // listening. Given that applying a rejudging will only create judgement and run events and
+        // that for these events contest state change events don't really matter, we will only check
+        // it once, here.
+        // We will also not check dependant object events in the loop, because if we apply a
+        // rejudging, the original judgings will already have triggered all dependant events.
```

Comment:
There's a typo: it should be "dependent".

Reply:
Thanks, seems I fail with this more often, as I found many places in the code that I now have fixed.